tests/Pinta.Effects.Tests/Mocks/MockChromeManager.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using Gio;
+using Gtk;
+using Mono.Addins.Localization;
+using Pinta.Core;
+
+namespace Pinta.Effects;
+
+internal sealed class MockChromeManager : IChromeManager
+{
+	public PointI LastCanvasCursorPoint { get => throw new NotImplementedException (); set => throw new NotImplementedException (); }
+
+	public Box? MainToolBar => throw new NotImplementedException ();
+
+	public Box ToolToolBar => throw new NotImplementedException ();
+
+	public Box ToolBox => throw new NotImplementedException ();
+
+	public Box StatusBar => throw new NotImplementedException ();
+
+	public IProgressDialog ProgressDialog => throw new NotImplementedException ();
+
+	public Menu AdjustmentsMenu => throw new NotImplementedException ();
+
+	public Menu EffectsMenu => throw new NotImplementedException ();
+
+	public Gtk.Application Application => throw new NotImplementedException ();
+
+	public Window MainWindow => throw new NotImplementedException ();
+
+	public Widget ImageTabsNotebook => throw new NotImplementedException ();
+
+	public bool MainWindowBusy { get => throw new NotImplementedException (); set => throw new NotImplementedException (); }
+
+	public event EventHandler? LastCanvasCursorPointChanged;
```

Comment:
There are lots of warnings here, but this might be solved by the other suggestion for reducing what's available in the interface